README.md

```diff
@@ -86,33 +105,36 @@ idrun
 
 ## To run frontend tests
 
-+ $ karma start
+```
+karma start
+```
 
 # Grunt Tasks
 
 ## Watch and compile front-end files
-+ $ grunt watch
-
-## Compile front-end files
-+ $ grunt compile
 
-# Pattern Library
+```
+grunt watch
+```
 
-A library of common patterns used accross the membership site is available at [membership.theguardian.com/patterns](https://membership.theguardian.com/patterns)
+## Compile front-end files
 
-When building new components break them down into fragments and include them in the pattern library.
+```
+grunt compile
+```
```

Comment:
worth putting the --dev flag in here? other tasks can be determined by examining the grunt file